cloud-controller-manager/civo/loadbalancer.go

```diff
@@ -189,7 +146,89 @@ func (l *loadbalancer) UpdateLoadBalancer(ctx context.Context, clusterName strin
 	}
 	lbuc.Backends = backends
 
-	ulb, err := l.client.civoClient.UpdateLoadBalancer(civolb.ID, &lbuc)
+	if ip := getReservedIP(service); ip != "" {
```

Comment:
I assumed that the `getReservedIP` function would return an IP object, not a string. 


We should consider renaming the method to `getReservedIPFromAnnotation` or similar. This would be clearer when reading the code. and comparing to the civoClient.FindIP call in the line below.